.github/renovate.json5

```
@@ -45,4 +45,6 @@
   "enabledManagers": [
     "tekton",
   ],
+  labels: ["auto-merge"],
```

[nit] Yes, json5 allows unquoted names but this still looks a bit unusual compared to other attributes. If we keep this attribute, let's please make it look consistent with others:
```suggestion
  "labels": ["auto-merge"],
```